crm/views.py

```diff
@@ -0,0 +1,12 @@
+
+
+from rest_framework import viewsets
+from .serializers import CompanySerializer
+
+
+# Import models from companies
+from companies.models import Company, CompanyAddress, CompanyCustomerResponsible, Group, CompanyContact, CompanyCustomerComment
+
+class ViewCompanies(viewsets.ModelViewSet):
+    queryset = Company.objects.all().order_by('name')
+    serializer_class = CompanySerializer
```

Comment:
If you could optimize the code to maybe follow these guidelines it would give us a standard way of how we are developing API.

[Styleguide api](https://github.com/HackSoftware/Django-Styleguide#apis--serializers)